File: gameserver/src/main/java/l2s/gameserver/network/authcomm/ReceivablePacket.java
package l2s.gameserver.network.authcomm;

import io.netty.buffer.ByteBuf;
import l2s.gameserver.network.authcomm.vertx.AuthServerConnection;

public abstract class ReceivablePacket extends l2s.commons.net.netty.ReceivablePacket<AuthServerConnection> {
    public ReceivablePacket(AuthServerConnection client, ByteBuf byteBuf) {
        super(client, byteBuf);
    }
}

File: gameserver/src/main/java/l2s/gameserver/network/authcomm/ReceivablePacket.kt
package l2s.gameserver.network.authcomm

import io.netty.buffer.ByteBuf
import l2s.commons.net.netty.ReceivablePacket
import l2s.gameserver.network.authcomm.vertx.AuthServerConnection

abstract class ReceivablePacket(
    client: AuthServerConnection,
    byteBuf: ByteBuf
) : ReceivablePacket<AuthServerConnection>(
    client,
    byteBuf
)